tinyagentos/vram_reservation.py

```
@@ -74,11 +74,16 @@ class VramReservationManager:
     def __init__(
         self,
         ttl_seconds: float = DEFAULT_RESERVATION_TTL_SECONDS,
+        probe: "Callable[[], tuple[int, int] | None] | None" = None,
```

Ruff (0.15.21)

[error] Undefined name `Callable`

(F821)